src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final int INDEX_OF_DAY = 0;
+    private static final int INDEX_OF_MONTH = 1;
+    private static final int INDEX_OF_YEAR = 2;
+    private static final int INDEX_OF_DATA = 0;
+    private static final int INDEX_OF_NAME = 1;
+    private static final int INDEX_OF_WORKED_HOURES = 2;
+    private static final int INDEX_OF_SALARY = 3;
+
+    public static String getSalaryInfo(String[] names, String[] data,
+                                       String dateFrom, String dateTo) {
+        StringBuilder stringBuilder = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo);
+        int[] salary = new int[names.length];
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
[DateTimeFormatter should be constant ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field)